test/createActions.test.ts

```diff
@@ -31,6 +31,25 @@ describe('test the generating actions function', () => {
     expect(generatedActions.requestSuccess(payload)).toEqual(expectedActions.requestSuccess);
   });
 
+  it('should return specified actions when use specified types on actions', () => {
+    const types = ['REQUEST', 'REQUEST_SUCCESS', 'REQUEST-FAILURE'];
+    const actionTypes = createActionTypes('TYPE', types);
+    const generatedActions = createActions<{
+      'request': { data: number },
+      'requestSuccess': { data: number },
+      'requestFailure': { data: number }
+    }>(actionTypes);
+    const payload = { data: 2 };
+    const expectedActions = {
+      request: { type: actionTypes.REQUEST, payload },
+      requestSuccess: { type: actionTypes.REQUEST_SUCCESS, payload },
+      requestFailure: { type: actionTypes['REQUEST-FAILURE'], payload },
+    };
+
+    expect(generatedActions.request(payload)).toEqual(expectedActions.request);
+    expect(generatedActions.requestSuccess(payload)).toEqual(expectedActions.requestSuccess);
+  });
```

Comment:
Should check the `generatedActions`'s `requestFailure` property as well, please help me add it to the above test case too.

Reply:
My pleasure.